packages/e2e-selenium/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare const webdriver: typeof import('selenium-webdriver');
```

Comment:
Did you intend for this package to be type-checked? (#18838)

See also (requirements): https://github.com/WordPress/gutenberg/blob/master/packages/README.md#typescript

Reply:
I simply added this to get intellisense support. But come to think of it, type-safe from day 1 isn't a bad idea.